netlogo-gui/src/main/app/codetab/CodeTab.scala

```diff
@@ -170,7 +174,7 @@ with MenuTab {
   def handle(e: AppEvents.SwitchedTabsEvent) {
     if (e.oldTab == this && dirty)
       compile()
-
+    
```

Comment:
Watch out for erroneous whitespace changes.  To prevent these, you can:

1. Not make them (okay, I'm just being smug)
2. Always check `git diff` / `git diff --staged` before committing
3. Configure your editor to trim trailing whitespace